packages/manager/apps/key-management-service/src/components/layout-helpers/Dashboard/GeneralInformationsTiles/BillingInformationsTile.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import {
+  OsdsButton,
+  OsdsChip,
+  OsdsDivider,
+  OsdsIcon,
+  OsdsLink,
+  OsdsText,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_LINK_REFERRER_POLICY,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { format, parseISO } from 'date-fns';
+import { OKMS } from '@/interface';
+import { useKMSServiceInfos } from '@/api/hooks/useKMSServiceInfos';
+
+type BillingInformationsTileProps = {
+  okmsData?: OKMS;
+};
+
+const BillingInformationsTile = ({
+  okmsData,
+}: BillingInformationsTileProps) => {
+  const { data: kmsService } = useKMSServiceInfos(okmsData);
+  const { t } = useTranslation('key-management-service/dashboard');
+
+  return (
+    <OsdsTile className="w-full h-fit flex-col" inline rounded>
+      <div className="flex flex-col w-full">
+        <OsdsText
+          size={ODS_TEXT_SIZE._400}
+          level={ODS_TEXT_LEVEL.heading}
+          color={ODS_THEME_COLOR_INTENT.text}
+        >
+          {t('billing_informations')}
+        </OsdsText>
+        <OsdsDivider separator />
+        <div className="flex flex-col mb-3">
+          <OsdsText
+            className="mb-4"
+            size={ODS_TEXT_SIZE._200}
+            level={ODS_TEXT_LEVEL.heading}
+            color={ODS_THEME_COLOR_INTENT.text}
+          >
+            {t('key_management_service_dashboard_field_label_creation_date')}
+          </OsdsText>
+          <div className="flex flex-row justify-between items-center">
+            <OsdsText
+              className="mb-4"
+              size={ODS_TEXT_SIZE._400}
+              level={ODS_TEXT_LEVEL.body}
+              color={ODS_THEME_COLOR_INTENT.default}
+            >
+              {kmsService &&
+                format(
+                  parseISO(
+                    kmsService?.data.billing.lifecycle.current.creationDate,
+                  ),
+                  'yyyy-MM-dd',
+                )}
```

Comment:
Should be managed regarding the locale defined for the user (with Intl.DateTimeFormat)